app/src/androidTest/java/mozilla/lockbox/uiTests/Navigator.kt

```diff
@@ -146,20 +145,16 @@ class Navigator {
     }
 
     fun checkAtWelcome() {
+        blockUntil(RouteStore.shared.routes, RouteAction.Login)
         welcome { exists() }
     }
 
     fun gotoItemList(goManually: Boolean = false) {
         if (goManually) {
             bypassOnboarding()
         } else {
-            Dispatcher.shared.dispatch(LifecycleAction.UseTestData)
-            log.info("blocking for the routes")
-            // block until onboarding
-            // tap skip
-            blockUntil(RouteStore.shared.routes, RouteAction.ItemList)
+            checkAtItemList()
         }
-        checkAtItemList()
```

Comment:
I think this is the root of where the problem needs to be fixed, but I don't think this will fix it.